src/generic/YoungTabs.jl

```diff
@@ -78,10 +78,47 @@ function Base.start(parts::AllParts{T}) where T<:Integer
     end
 end
 
+const _numPartsTable = Dict{Int, Int}(0 => 1, 1 => 1, 2 => 2)
+const _numPartsTableBig = Dict{Int, BigInt}()
+
+doc"""
+    _numpart(n::Int)
+> Returns the number of all distinct integer partitions of `n`. The function
+> uses Euler pentagonal number theorem for recursive formula. For more details
+> see OEIS sequence [A000041](http://oeis.org/A000041). Note that
+> `noPartitions(0) = 1` by convention.
```

Comment:
`_numpart(0) = 1`?

Reply:
Yeah, the leftovers spott the function, I'm fixing it and adding tests as well